src/java.sql.rowset/share/classes/javax/sql/rowset/serial/SerialBlob.java

```diff
@@ -323,33 +322,33 @@ public int setBytes(long pos, byte[] bytes)
     }
 
     /**
-     * Writes all or part of the given <code>byte</code> array to the
-     * <code>BLOB</code> value that this <code>Blob</code> object represents
+     * Writes all or part of the given {@code byte} array to the
+     * {@code BLOB} value that this {@code Blob} object represents
      * and returns the number of bytes written.
-     * Writing starts at position <code>pos</code> in the <code>BLOB</code>
-     * value; <i>len</i> bytes from the given byte array are written.
+     * Writing starts at position {@code pos} in the {@code BLOB}
+     * value; {@code length} bytes from the given byte array are written.
      *
-     * @param pos the position in the <code>BLOB</code> object at which
-     *     to start writing. The first position is <code>1</code>;
-     *     must not be less than <code>1</code> nor greater than
-     *     the length of this <code>SerialBlob</code> object.
-     * @param bytes the array of bytes to be written to the <code>BLOB</code>
+     * @param pos the position in the {@code BLOB} object at which
+     *     to start writing. The first position is {@code 1};
+     *     must not be less than {@code 1} nor greater than
+     *     the length+1 of this {@code SerialBlob} object.
+     * @param bytes the array of bytes to be written to the {@code BLOB}
      *     value
-     * @param offset the offset in the <code>byte</code> array at which
-     *     to start reading the bytes. The first offset position is
-     *     <code>0</code>; must not be less than <code>0</code> nor greater
-     *     than the length of the <code>byte</code> array
+     * @param offset the offset into the array {@code bytes} at which
```

Comment:
Please change all occurrences of  `{@code bytes}` to `{@code byte}s` as this was caught as part of the CSR review.